certificate-policy.md

```diff
@@ -1149,73 +1138,81 @@ DSA: Although FIPS 800-57 says that domain parameters may be made available at s
-
-    e. The CA documents that the Applicant's application cannot be patched or replaced without substantial economic outlay;
-
-    f. The CA signs the Subscriber Certificate on or before June 30, 2016; and
-
-    g. The notBefore field in the Subscriber Certificate has a date on or before June 30, 2016.
+4. Certificates issued solely for the purpose of testing products with Certificates issued by the Root CA.
 
 
 ## 6.2 Private Key Protection and Cryptographic Module Engineering Controls
 The CA SHALL implement physical and logical safeguards to prevent unauthorized certificate issuance. Protection of the CA Private Key outside the validated system or device specified above MUST consist of physical security, encryption, or a combination of both, implemented in a manner that prevents disclosure of the Private Key. The CA SHALL encrypt its Private Key with an algorithm and key-length that, according to the state of the art, are capable of withstanding cryptanalytic attacks for the residual life of the encrypted key or key part.
 
 ### 6.2.1 Cryptographic module standards and controls
+The relevant standard for cryptographic modules is Security Requirements for Cryptographic Modules [FIPS 140-2]. Cryptographic modules shall be validated to a FIPS 140 level identified in this section.
+
+- Cryptographic modules for CAs and OCSP responders SHALL be hardware modules validated as meeting FIPS 140-2 Level 3 or above
+- Cryptographic modules for Subscribers SHALL be FIPS 140-2 Level 1 or above
+
 
 ### 6.2.2 Private key (n out of m) multi-person control
+For CA Key Pairs that are either (i) used as Root CA Key Pairs or (ii) Key Pairs generated for a subordinate CA that is not the operator of the Root CA or an Affiliate of the Root CA, or (iii) Key Pairs for the operator of the Root CA or an Affiliate of the Root CA:
+
+- A single person shall not be permitted to activate or access any cryptographic module that contains the complete CA private signing key. 
+- CA signature keys may be backed up only under at least two-person control. 
+- Access to CA signing keys backed up for disaster recovery shall be under at least two-person control. 
+- The names of the parties used for two-person control shall be maintained on a list that shall be made available for inspection during by Qualified Auditors.
 
 ### 6.2.3 Private key escrow
+For CA Key Pairs that are either (i) used as Root CA Key Pairs or (ii) Key Pairs generated for a subordinate CA that is not the operator of the Root CA or an Affiliate of the Root CA, or (iii) Key Pairs for the operator of the Root CA or an Affiliate of the Root CA:
+- The CA private keys SHALL never be escrowed
 
 ### 6.2.4 Private key backup
+For CA Key Pairs that are either (i) used as Root CA Key Pairs or (ii) Key Pairs generated for a subordinate CA that is not the operator of the Root CA or an Affiliate of the Root CA, or (iii) Key Pairs for the operator of the Root CA or an Affiliate of the Root CA:
```

Comment:
drop first sentence of 6.2.4 - the rest applies to all CAs operating under this CP